lib/models/movie.dart

```diff
@@ -0,0 +1,55 @@
+class Movie {
+  final String title;
+  final String originalTitle;
+  final String overview;
+  final String releaseDate;
+  final double voteAverage;
+  final List<String> genres;
+  final String pathToBackdropImg;
+  final String pathToPosterImg;
+
+  const Movie(
+      this.title,
+      this.originalTitle,
+      this.overview,
+      this.releaseDate,
+      this.voteAverage,
+      this.genres,
+      this.pathToBackdropImg,
+      this.pathToPosterImg);
+}
+
+abstract class MovieFactory {
+  Movie getMovie();
+
+  const MovieFactory();
+}
+
+class MovieMockUpFactory extends MovieFactory {
+  static const title = 'The Super Mario Bros. Movie';
+  static const originalTitle = 'The Super Mario Bros. Movie';
+  static const overview =
+      'While working underground to fix a water main, Brooklyn '
+      'plumbers—and brothers—Mario and Luigi are transported down a mysterious '
+      'pipe and wander into a magical new world. But when the brothers are separated, '
+      'Mario embarks on an epic quest to find Luigi.';
+  static const releaseDate = '2023-04-05';
+  static const voteAverage = 7.8;
+  static const genres = [
+    'Animation',
+    'Family',
+    'Adventure',
+    'Fantasy',
+    'Comedy'
+  ];
+  static const pathToBackdropImg = 'images/SuperMarioBackdrop.jpg';
+  static const pathToPosterImg = 'images/SuperMarioPoster.jpg';
+
+  @override
+  Movie getMovie() {
+    return const Movie(title, originalTitle, overview, releaseDate, voteAverage,
+        genres, pathToBackdropImg, pathToPosterImg,);
+  }
```

Comment:
You add the trailing comma but forgot to format the file

Reply:
fixed